README.md

```diff
@@ -789,6 +790,8 @@ Supported `temp` values: 1-65279 mired. Actual color temperature supported by en
 
 - Mix of warm white intensity and cold white intensity forms color temperature.
 
+- Duration property within cmd.color.start_transition command is optional and means seconds.
```

Comment:
it should be mentioned that above `maxSeconds` seconds will not be handled and the value will be cut into full minutes, with example, sending 128 (seconds, == 2 min 8 seconds) will result in adapter setting only 2 min and not 2m 8s

Reply:
Extended description as much as I can :D 
If someone is able to describe it better, then write it down please so I could update it.